src/operator/integrate.rs

```diff
@@ -5,6 +5,46 @@ use crate::{
 };
 use std::ops::Add;
 
+/// Struct returned by the [`Stream::integrate`] operation.
+///
+/// This struct bundles the four output streams produced by the `integrate`
+/// method:
+///
+/// * `current` - the current value of the integral, i.e., the sum of all values
+///   in the stream since the last `clock_start` including the current clock
+///   cycle.
+/// * `delayed` - the previous value of the integral, i.e., the sum of all
+///   values in the stream since the last `clock_start` up to the previous clock
+///   cycle.
+/// * `export` - stream exported to the parent circuit that contains the final
+///   value of the integral at the end of the nested clock epoch.
+/// * `delta` - value added to the integral at the last clock cycle.  This is
```

Comment:
that's a bit strange.
This could really be done for all operators if you find it convenient.

Reply:
For unary operators it should be called "input"; delta is very confusing, it suggests that something has been done to the input.

Reply:
It's the delta between delayed and current.

Reply:
which is the current input, isn't it?

Reply:
But yeah, it can be confusing, I'll rename